testing/solomachine.go

```diff
@@ -129,6 +132,17 @@ func (solo *Solomachine) CreateClient(chain *TestChain) string {
 	return clientID
 }
 
+// UpdateClient sends a MsgUpdateClient to the provided chain and updates the given clientID.
+func (solo *Solomachine) UpdateClient(chain *TestChain, clientID string) {
```

Comment:
We could add a `newDiversifier` argument here but I guess its not required at the moment. 
Could be a future improvement